wear/src/main/java/com/example/wear/snippets/rotary/Rotary.kt

```diff
@@ -0,0 +1,219 @@
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.input.pointer.pointerInteropFilter
+import androidx.compose.ui.input.rotary.onRotaryScrollEvent
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.foundation.ExperimentalWearFoundationApi
+import androidx.wear.compose.foundation.lazy.ScalingLazyColumn
+import androidx.wear.compose.foundation.lazy.rememberScalingLazyListState
+import androidx.wear.compose.foundation.rememberActiveFocusRequester
+import androidx.wear.compose.material.Chip
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.Picker
+import androidx.wear.compose.material.PositionIndicator
+import androidx.wear.compose.material.Scaffold
+import androidx.wear.compose.material.Text
+import androidx.wear.compose.material.rememberPickerState
+import androidx.wear.compose.ui.tooling.preview.WearPreviewDevices
+import androidx.wear.compose.ui.tooling.preview.WearPreviewFontScales
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalWearFoundationApi::class)
+@Composable
+fun ScrollableScreen() {
+    // This sample doesn't add a Time Text at the top of the screen.
+    // If using Time Text, add padding to ensure content does not overlap with Time Text.
+    // [START android_wear_rotary_input]
+    val listState = rememberScalingLazyListState()
+    Scaffold(
+        positionIndicator = {
+            PositionIndicator(scalingLazyListState = listState)
+        }
+    ) {
+
+        val focusRequester = rememberActiveFocusRequester()
+        val coroutineScope = rememberCoroutineScope()
+
+        ScalingLazyColumn(
+            modifier = Modifier
+                .onRotaryScrollEvent {
+                    coroutineScope.launch {
+                        listState.scrollBy(it.verticalScrollPixels)
+                        listState.animateScrollBy(0f)
+                    }
+                    true
+                }
+                .focusRequester(focusRequester)
+                .focusable()
+                .fillMaxSize(),
+            state = listState
+        ) {
+            // Content goes here
+            // [START_EXCLUDE]
+            items(count = 5) {
+                Chip(onClick = { }, label = { Text("Item #$it") })
+            }
+            // [END_EXCLUDE]
+        }
+    }
+    // [END android_wear_rotary_input]
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun TimePicker() {
+    val textStyle = MaterialTheme.typography.display1
+
+    // [START android_wear_rotary_input_picker]
+    var selectedColumn by remember { mutableIntStateOf(0) }
+
+    val focusRequester1 = remember { FocusRequester() }
```

Comment:
Are focusRequester1 and 2 just one per `Picker` component? Wondering if there is a better name...?

Reply:
good point, renamed to  `minutesFocusRequester` and `hoursFocusRequester`